### PR TITLE
OPSEXP-1746: add an helm-unittest action

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1746

installs `helm-unittest` plugin and runs tests if a `tests` folder exists

### CODE REVIEW

File: .github/actions/helm-unit-tests/action.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - name: Run Helm Unit tests
+      shell: bash
+      working-directory: ${{ inputs.chart-dir }}
+      if: ${{ inputs.chart-type }} != 'library'
+      run: |
+        set -e
+        if [ -d tests ]; then echo "${{ inputs.chart-dir }}: Unit tests"
+          helm plugin install https://github.com/vbehar/helm3-unittest
+          helm dep up
+          helm unittest --color .
```
Comment (gionn) [inline .github/actions/helm-unit-tests/action.yml:31]: you need to cd chart-dir before?

File: .github/actions/helm-unit-tests/action.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+name: Helm Unit tests
+description: >
+  Look for helm-unittest "test" folder and run the tests
+inputs:
+  plugin_version:
+    description: version of the helm-unittest plugin to install
+    required: false
+    default: v1.0.16
```
Comment (gionn) [inline .github/actions/helm-unit-tests/action.yml:9]: if not yet implemented just remove this input or fix the usage

**Review by gionn — CHANGES_REQUESTED** [review]

File: .github/actions/helm-unit-tests/action.yml (hunk 1 above)
Comment (alxgomz) [reply .github/actions/helm-unit-tests/action.yml:31]: it's using the `working-directory` prop

**Review by gionn — COMMENTED** [review]
add an entry into the main README so it's more easy to discover it

**Review by gionn — APPROVED** [review]

**Pull request merged**